CruCentralCoast/Events/EventDetailsVC.swift

```diff
@@ -0,0 +1,81 @@
+//
+//  EventDetailsVC.swift
+//  CruCentralCoast
+//
+//  Created by Cam Stocker on 4/25/18.
+//  Copyright © 2018 Landon Gerrits. All rights reserved.
+//
+
+import UIKit
+
+struct EventCellParameters {
+    let titleLabel : String
+    let date : String
+    let location : String
+    let description : String
+}
+
+class EventDetailsVC: UIViewController {
+
+    @IBOutlet weak var imageView: UIImageView!
+    @IBOutlet weak var dateLabel: UILabel!
+    @IBOutlet weak var eventTitleLabel: UILabel!
+    @IBOutlet weak var locationLabel: UILabel!
+    @IBOutlet weak var descriptionText: UILabel!
+    
+    @IBOutlet weak var widthConstraint: NSLayoutConstraint!
+    
+    var titleLabel : String?
+    var date: String?
+    var location: String?
+    var desc: String?
+    
+    
+    @IBAction func dismissDetail(_ sender: Any) {
+        self.dismiss(animated: true, completion: nil)
+    }
+    
+    @IBAction func FBButton(_ sender: Any) {
```

Comment:
FBButton sounds like the name of an @IBOutlet rather than an @IBAction. Maybe something like "facebookButtonPressed" would make more sense